### DISCUSSION TITLE
phase field simulation for two-phase sintering

### DISCUSSION THREAD

**Discussion by Wolke926** [idaholab/moose#24536] [Q&A Modules: Phase field]
![image](https://github.com/idaholab/moose/assets/120539827/8f795730-1742-496d-aeb0-247dc7900288)

![Screen Shot 2023-05-30 at 12 03 47 PM](https://github.com/idaholab/moose/assets/120539827/94288291-e354-4555-90ec-dec8e5fbfc51)
![Screen Shot 2023-05-30 at 12 05 18 PM](https://github.com/idaholab/moose/assets/120539827/790b4e28-798d-486a-b697-26b76aab2b1a)
![Screen Shot 2023-05-30 at 12 06 35 PM](https://github.com/idaholab/moose/assets/120539827/0a41f859-5681-40cc-899b-17f9b10cac3f)

**Comment by lindsayad** [comment]
This is unreadable for me. See attached
<img width="937" alt="Screen Shot 2023-05-30 at 11 24 51 AM" src="https://github.com/idaholab/moose/assets/10248304/c7850fad-76dc-46a2-b079-3083057ab60f">

**Reply by GiudGiud** [reply]
The variable names are all different?
I imagine c_a is Chi_a
Is eta_s  phi_alpha ?

The advection terms seem fine for now. Once you calculate u using the Navier Stokes module, you will need to make sure that the discretization schemes for u in the NS equations and these equations are rigorously the same.

**Reply by Wolke926** [reply]
Yes, chi_A is the c_A, the concentration, and the eta_s is phi_alpha. Sorry for the misunderstanding.
